### PR TITLE
config-solaris.md: fix info

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: config-solaris.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The property should consist of a comma-separated privilege set specification as
 ## <a name="configSolarisMaxShmMemory" />maxShmMemory
 The maximum amount of shared memory allowed for this application container.
 A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
-Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m] man page.
+Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m_2] man page.
```
Comment (hqhq) [inline config-solaris.md]: I got 404 error from this link, I think the right link is: https://docs.oracle.com/cd/E36784_01/html/E36871/zonecfg-1m.html

And I think it makes more sense to use alias `zonecfg.1m` for this link.

Comment (q384566678) [inline config-solaris.md]: updated, thanks.

Comment (wking) [inline config-solaris.md]: > And I think it makes more sense to use alias `zonecfg.1m` for this link.

You need to use `zonecfg.1m_2` to follow [our policy][1] and avoid conflicts with [the `config.md` entry][2].

[1]: https://github.com/opencontainers/runtime-spec/blame/101e6f9d5c32579c245d8f45f4bdcd87f9a40cf9/style.md#L35
[2]: https://github.com/opencontainers/runtime-spec/blame/101e6f9d5c32579c245d8f45f4bdcd87f9a40cf9/config.md#L838

**Comment by crosbymichael** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**